tests/modules/programs/mbsync/mbsync.nix

```
@@ -21,7 +21,7 @@ with lib;
     accounts.email.accounts = {
       "hm@example.com".mbsync = {
         enable = true;
-        extraConfig.account.SSLVersions = [ "TLSv1.3" "TLSv1.2" ];
+        extraConfig.account.TLSVersions = [ "TLSv1.3" "TLSv1.2" ];
```

```suggestion
        extraConfig.account.TLSVersions = [ "TLSv1.3" "TLSv1.2" ];
        extraConfig.account.TLSVersions = [ "+1.3" "+1.2" ];
```

Needed in addition to https://github.com/nix-community/home-manager/pull/5918#discussion_r1816360564